clustered/integration-test/src/test/java/org/ehcache/clustered/writebehind/BasicClusteredWriteBehindTest.java

```diff
@@ -127,4 +133,40 @@ public void testClusteredWriteBehindLoading() throws Exception {
 
     doThreadDump = false;
   }
+
+  @Test
+  public void testBasicClusteredWriteBehindWithFailure()  {
+    cache = null;
+    cacheManager.close();
+    cacheManager = null;
+    cacheManager = createCacheManagerWithLoaderWriterWithFailure(CLUSTER.getConnectionURI());
+    cache = cacheManager.getCache(testName.getMethodName(), Long.class, String.class);
+    long counter;
+    for (counter = 1; counter <= 4; counter++) {
```

Comment:
I think all threee of these loops will be easier to follow if you just unroll them.